bedrock/firefox/views.py

```diff
@@ -93,7 +93,7 @@ def get_context_data(self, **kwargs):
 @require_GET
 def stub_attribution_code(request):
     """Return a JSON response containing the HMAC signed stub attribution value"""
-    if not request.is_ajax():
```

Comment:
oh! `is_ajax()` is gone?

Reply:
Sure is. Makes sense. I did always think it was weird that it relied on a jQuery convention.